docker/sandbox/Dockerfile-php.template

```diff
@@ -1,8 +1,8 @@
 FROM artifactory.wikia-inc.com/sus/php-wikia-base:eb11f0c
 
-ENV WIKIA_DATACENTER="sjc"
-ENV WIKIA_ENVIRONMENT="sandbox"
-ENV HOSTNAME_OVERRIDE="sandbox-sus2"
+ENV WIKIA_DATACENTER="${SANDBOX_DATACENTER}"
+ENV WIKIA_ENVIRONMENT="${SANDBOX_ENVIRONMENT}"
```

Comment:
I think we can keep `sandbox` hardcoded here. We want to deploy sandbox here after all, right?

Reply:
true, however, I've wanted to keep vars in sync with the ones from Jenkinsfile. "sandbox" is hardcoded there